docker/regtest/scripts/setup.sh

```
@@ -124,6 +142,40 @@ echo "Opened channels to CLN"
 waitForClnChannel "lightning-cli"
 waitForClnChannel "lightning-cli --regtest --lightning-dir /root/.lightning2"
 
+# ASP setup
+echo "Funding ASP"
+
+arkd wallet create --password ark > /dev/null
+waitForArkdToSync
+
+bitcoin-cli sendtoaddress $(arkd wallet address) 25 > /dev/null
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)